openstack-details

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+utility_container=`lxc-ls | grep utility`
+
+echo "============================================================="
+echo "openrc"
+echo "============================================================="
+set -x
+lxc-attach -n $utility_container cat /root/openrc | tee ~/.ostrich/openrc
```

Comment:
It finds the utility container, attaches to it, sources the openrc, and executes the command you provide in that environment.